src/js/components/RadioButtonGroup/doc.js

```diff
@@ -61,3 +61,11 @@ export const doc = RadioButtonGroup => {
 
   return DocumentedRadioButtonGroup;
 };
+
+export const themeDoc = {
+  'radioButtonGroup.container': {
+    description: 'Any valid Box props for the RadioButtonGroup container.',
+    type: 'object',
+    defaultValue: "{ align: 'start', pad: 'small' , ...}",
```

Comment:
Are you sure this is the default value? I'm not sure it is.

Reply:
used the same description for select 
https://v2.grommet.io/select#select.options.container

```
  'select.options.container': {
    description: 'Any valid Box prop for the options container.',
    type: 'object',
    defaultValue: "{ align: 'start', pad: 'small' }",
  },
```

my assumption is that the description on the select page is not default values either, just showing values that you can pass into select.options.container.

Reply:
Please put the correct default values for the RBG, and take a look at how the Select entry could be fixed as well.

Reply:
changed default to an empty object